src/app.rs

```diff
@@ -220,14 +306,22 @@ impl Context {
         port: Option<u32>,
         table: Option<String>,
     ) -> Result<Context, DyneinConfigError> {
+        let config = load_or_touch_config_file(true)?;
+        let retry = match &config.retry {
```

Comment:
Because of the addition of validation, if an `ExponentialBuilder` is generated in batch_write_item, the interfaces will not match, so the `ExponentialBuilder` is generated here.

This also helps reduce overhead during `batch_write_item`.

Reply:
I think this is a reasonable improvement. Thank you for proactively implementing this.